__tests__/event_manager.test.ts

```diff
@@ -0,0 +1,28 @@
+import Ain from '../src/ain';
+
+const { test_event_handler_node } = require('./test_data');
+const delayMs = (time) => new Promise(resolve => setTimeout(resolve, time));
+
+jest.setTimeout(60000);
+
+describe('Event Handler', function() {
+  let ain = new Ain(test_event_handler_node);
+
+  beforeAll(async () => {
+    await ain.em.connect();
+  });
+
+  afterAll(() => {
+    ain.em.disconnect();
+  });
+
+  it('Subscribe BLOCK_FINALIZED', async () => {
+    const callback = jest.fn();
+    const subscription = ain.em.subscribe('BLOCK_FINALIZED', {});
+    subscription.once('data', (data) => {
```

Comment:
I think this callback function itself can be passed to subscribe() e.g.
```
const filterId = ain.em.subscribe('BLOCK_FINALIZED', 'once', (data) => {
  ...
});
```
and the id of the created filter can be returned instead of the event emitter?
For testing, I think the emitter can be obtained by `ain.em.eventEmitter[filterId]` (if eventEmitter[] is moved to event callback manager, then this should be updated accordingly.)
what do you think?